ql/src/test/results/clientpositive/llap/alter_change_db_location.q.out

```diff
@@ -11,7 +11,7 @@ PREHOOK: Input: database:newdb
 POSTHOOK: query: describe database extended newDB
 POSTHOOK: type: DESCDATABASE
 POSTHOOK: Input: database:newdb
-newdb		location/in/test		hive_test_user	USER			
+#### A masked pattern was here ####
```

Comment:
The managedlocation is like "file:/home/robbie/hive/itests/qtest/target/localfs/warehouse/newdb.db" which matches "file:" in QOutProcessor.maskIfContains.

Reply:
I see, I created a followup ticket for this: HIVE-25473 (in order to get useful info back)

Reply:
Actually the original output is like:
newdb		location/in/test	file:/home/robbie/hive/itests/qtest/target/localfs/warehouse/newdb.db	hive_test_user	USER

The managedlocation is not empty. Because of the pattern "file:/", QOutProcessor masks the whole line.